styles/_settings.scss

```diff
@@ -27,9 +27,9 @@ $cloudy-blue: #aadad1;
 $light-grey-blue: #8db6ca;
 $warm-pink: #f65e6e;
 
-$link-green: #bcda59;
-$link-blue: #2f7fab;
-$link-pink: $warm-pink;
+$qual-soybeans: #bcda59;
+$qual-soybeancake: #2f7fab;
+$qual-soybeanoil: $warm-pink;
```

Comment:
- I want explicit semantic aliases for colors used in this recolor feature (they are used in 3 different places)
- `$qual-soybeanoil: $warm-pink;` is temporary anyways, as this Commodity recoloring will not make it to the final product (it's just used as a test in the DB)